build.proj

```diff
@@ -69,6 +69,8 @@
 
     <ItemGroup>
       <PkgProject Include="pkg\Microsoft.ML.Auto\Microsoft.ML.Auto.nupkgproj" />
+      <PkgProject Include="pkg\Microsoft.ML.Auto\Microsoft.ML.Auto.symbols.nupkgproj" />
+      <PkgProject Include="pkg\mlnet\mlnet.nupkgproj" />
```

Comment:
How do we want to handle the symbols for `mlnet.dll`? Do we want to publish them to the Microsoft symbols server (so people [including us] can debug the released app)?